### PR TITLE
Thycotic Integration

### PR DESCRIPTION
**Pull request by AndyNikolaev**
Fixes #

## Proposed Changes

  - Added functionality for integration with Thycotic Secret Server
  -
  -

### CODE REVIEW

**Comment by AndyNikolaev** [comment]
Hey @preetikr,
Could you take a look and evaluate this solution.
Added DataConnector, Workbook and Dashboard.
Thank you.

File: DataConnectors/ThycoticCEF.json [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+                    "read": true,
+                    "write": true,
+                    "delete": true
+                }
+            }
+        ],
+        "customs": [
+            {
+                "name": "Thycotic Secret Server",
+                "description": "must be configured to export logs via Syslog \n\n   [Learn more about configure Secret Server](https://docs.thycotic.com/ss/10.8.0/events-and-alerts/secure-syslog-cef)"
```
Comment (wsmelton) [inline DataConnectors/ThycoticCEF.json:58]: We are adding a short link to this page so the product version does not have to be used here. Please update this to: `https://updates.thycotic.net/links.ashx?syslog`.

This link should be live by the end of the week.

Comment (wsmelton) [reply DataConnectors/ThycoticCEF.json]: Sorry correction on link (forgot we are moving to a new system) 😬 

`https://thy.center/ss/link/syslog`

**Comment by AndyNikolaev** [comment]
@wsmelton What needs to be done to approve?

File: DataConnectors/ThycoticCEF.json [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+                    "read": true,
+                    "write": true,
+                    "delete": true
+                }
+            }
+        ],
+        "customs": [
+            {
+                "name": "Thycotic Secret Server",
+                "description": "must be configured to export logs via Syslog \n\n   [Learn more about configure Secret Server](https://thycotic.force.com/support/s/documents)"
```
Comment (wsmelton) [inline DataConnectors/ThycoticCEF.json:58]: This is no longer the location of our docs.

We have created a short URL that points directly to the syslog configuration now: `https://thy.center/ss/link/syslog`

Comment (AndyNikolaev) [reply DataConnectors/ThycoticCEF.json]: Changed link

**Comment by AndyNikolaev** [comment]
@wsmelton Required changes added

**Comment by AndyNikolaev** [comment]
@wsmelton What needs to be done to approve? :-)

**Comment by wsmelton** [comment]
The assigned Azure team member is responsible for the approval of the PR.

**Comment by wsmelton** [comment]
@tianderturpijn @preetikr please reopen this PR as it was closed by someone referencing it incorrectly in PR #1120

**Comment by AndyNikolaev** [comment]
@wsmelton 
can i create a new one pull request?

**Comment by wsmelton** [comment]
@AndyNikolaev I guess you might as well, seems none of the maintainers are going to reopen it.

**Comment by AndyNikolaev** [comment]
@wsmelton

**Pull request closed without merging**